libs/sdk-ui-ext/src/internal/components/dialogs/embedInsightDialog/EmbedInsightDialog.tsx

```diff
@@ -0,0 +1,174 @@
+    }, [codeOption, insight, settings, backend, colorPalette, codeLang]);
+
+    const onCodeOptionChange = useCallback((codeOpt: IOptionsByDefinition) => {
+        setCodeOption(codeOpt);
+    }, []);
+
+    const documentationLink = useMemo(
+        () => getLinkToPropertiesDocumentation(codeType, insight),
+        [codeType, insight],
+    );
+
+    return (
+        <IntlWrapper locale={locale}>
+            <ModalOverlay>
+                <EmbedInsightDialogBase
+                    code={code}
+                    codeLanguage={codeLang}
+                    codeOption={codeOption}
+                    propertiesLink={documentationLink}
+                    integrationDocLink={integrationDocLink}
+                    onClose={onClose}
+                    onCopyCode={onCopyCode}
+                    onCodeLanguageChange={onLanguageChange}
+                    onCodeOptionChange={onCodeOptionChange}
+                />
+            </ModalOverlay>
+        </IntlWrapper>
+    );
+};
+
+const getLinkToPropertiesDocumentation = (
+    codeType: InsightCodeType,
+    insight: IInsight,
+): string | undefined => {
+    if (codeType === "definition") {
+        const meta = FullVisualizationCatalog.forInsight(insight).getMeta();
+        if (meta?.documentationUrl) {
+            return `${meta?.documentationUrl}#properties`;
+        }
+    }
+
+    return INSIGHT_VIEW_PROPERTIES_LINK;
+};
+
+const generateCodeByReference = (
+    codeOption: IOptionsByReference,
+    insight: IInsight,
+    settings: IUserWorkspaceSettings,
+    backend: IAnalyticalBackend,
+    colorPalette: IColorPalette,
+    codeLang: CodeLanguageType,
+) => {
+    const height = getHeightWithUnitsForEmbedCode(codeOption);
+    return insightViewCodeGenerator(insight, {
+        context: {
+            settings: settings,
+            backend: backend,
+            colorPalette: colorPalette,
+        },
+        language: codeLang,
+        height: height,
+        omitChartProps: codeOption.displayTitle ? [] : ["showTitle"],
+    });
+};
+
+const generateCodeByDefinition = (
+    codeOption: IOptionsByDefinition,
+    insight: IInsight,
+    settings: IUserWorkspaceSettings,
+    backend: IAnalyticalBackend,
+    colorPalette: IColorPalette,
+    codeLang: CodeLanguageType,
+) => {
+    const height = getHeightWithUnitsForEmbedCode(codeOption);
+
+    const descriptor = FullVisualizationCatalog.forInsight(insight);
+    return descriptor.getEmbeddingCode?.(insight, {
+        context: {
+            settings: settings,
+            backend: backend,
+            colorPalette: colorPalette,
+        },
+        language: codeLang,
+        height: height,
+        omitChartProps: codeOption.includeConfiguration ? [] : ["config"],
+    });
+};
+
+const ModalOverlay: React.FC = (props) => {
+    const { children } = props;
+    return (
+        <Overlay
+            alignPoints={[
```

Comment:
Please move the alignPoints to a constant outside of the component, as we do here for example: https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/AllItemsFilteredMessage.tsx#L10

Reply:
Fixed